panel/chat/feed.py

```diff
@@ -470,13 +473,25 @@ async def _schedule_placeholder(
             return
 
         start = asyncio.get_event_loop().time()
-        while not self._callback_state == CallbackState.IDLE and num_entries == len(self._chat_log):
+        while not task.done() and num_entries == len(self._chat_log):
             duration = asyncio.get_event_loop().time() - start
             if duration > self.placeholder_threshold:
                 self.append(self._placeholder)
                 return
             await asyncio.sleep(0.28)
```

Comment:
Any specific reason for this timeout?

```suggestion
            await asyncio.sleep(0.28)
```

Reply:
So it doesn't completely take over the event loop.

Reply:
Sure, but why 0.28s? Seems oddly specific and also quite long.

Reply:
It's one of my favorite numbers xD

No objections to changing it to like 0.1.

Reply:
Haha okay, yes something like 100ms seems more reasonable. The loop body should only take a few hundred milliseconds to execute.